### ISSUE TITLE
Module 03: Design and implement a release strategy

### ISSUE THREAD

**Issue by manishrps** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#596]
### Contact Details

manish.kumar@spektrasystems.com

### What happened?

Exercise 3: Configure the release pipeline
Task: 01
Step: 25

Description of issue: we are not able to get web page (E-commerce website) loads successfully in a new web browser tab

Repro steps:
1. Switch to the Azure portal interface, navigate to the resource group az400m04l09-RG, in the list of resources, click the DevTest web app, on the web app blade, click Browse, and verify that the web page (E-commerce website) loads successfully in a new web browser tab. 
![image](https://github.com/user-attachments/assets/5e1169d0-e897-4c74-a102-58c66e2488e3)

1.
1.

### Lab

Control Deployments using Release Gates

### Relevant screenshots

paste here 😉
![image](https://github.com/user-attachments/assets/f22c6596-9dc6-4ca8-9370-fb9c0e845a75)

### Do you want to help us? 👏

- [ ] Create a Pull Request by following this project's [Contribution Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#how-can-i-collaborate)

Could you please help me: @seesharprun @drahamim @smartdevskill @maartenvandiemen

**Comment by yashints** [comment]
Unfortunately this won't help us to identify the issue as we need more information and the lab environment is not available to us

**Comment by maartenvandiemen** [comment]
@manishrps I just followed the steps from the labs and I was able to successfully start the site. 
![image](https://github.com/user-attachments/assets/5a00ef8a-098c-4fde-a0e3-a7238b8c996c)

I found some other minor things but I will create a seperate PR for this.

I can reproduce the issue by removing the values as application settings. (Exercise 3, Task 1, Step 18). I think this is your issue.
